app/build.gradle

```diff
@@ -27,7 +28,13 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'androidx.recyclerview:recyclerview:1.1.0-beta04'
+    implementation "androidx.room:room-runtime:$roomVersion"
+
     testImplementation 'junit:junit:4.12'
+    testImplementation "androidx.room:room-testing:$roomVersion"
+
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+
+    kapt "androidx.room:room-compiler:$roomVersion"
```

Comment:
`kapt` should be below its parent library, in our case under `implementation "androidx.room:room-runtime:$roomVersion"`